lua/telescope/health.lua

```diff
@@ -100,7 +102,7 @@ M.check = function()
           )
         end
       else
-        local eol = version:find "\n"
+        local eol = version:find "\n" or 1
         ok(("%s: found %s"):format(package.name, version:sub(0, eol - 1) or "(unknown version)"))
```

Comment:
```suggestion
        local eol = version:find "\n"
        local ver = eol and version:sub(0, eol - 1) or "(unknown version)"
        ok(("%s: found %s"):format(package.name, ver))
```

> FIXME: In Windows environment, io.popen(binary .. " --version") does not return any outputs. How can I fix it?

Not really a great solution but this should at least closer be to what this bit of code is trying to accomplish regarding failing to get version number.